Resources/Private/Fusion/Prototypes/TrackingCode.GA.fusion

```diff
@@ -17,7 +17,16 @@ prototype(Neos.GoogleAnalytics:TrackingCode.GA) < prototype(Neos.GoogleAnalytics
             create = 'window.dataLayer = window.dataLayer || [];'
             push = 'function gtag(){dataLayer.push(arguments);}'
             date = 'gtag("js", new Date());'
-            config = ${'gtag("config", "' + props.trackingId + '", ' + props.parameters + ');'}
+            config = Neos.Fusion:Case {
```

Comment:
How about a join instead, so it would also be easier to adjust if needed?

```suggestion
            config = Neos.Fusion:Join {
                config = 'config'
                trackingId = ${props.trackingId}
                parameters = ${props.parameters}
                parameters.@if.set = ${props.parameters && props.parameters != 'null' && props.parameters != '[]'}
                @glue = ', '
                @process.wrap = ${'gtag(' + value + ');'}
            }
```

Reply:
Thats is way more elegant. Thanks